bin/helg

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env node
-var helg = require('../');
-console.log(helg.ere());
```

Comment:
Not sure about this change. Why move it? IIRC it's a common code convention in node to place executable "binaries" inside the `bin/` folder.